bip-0009/assignments.mediawiki

```diff
@@ -18,11 +18,11 @@ State can be defined, active, failed. Dates are in UTC.
 | csv
 | 0
 | 2016-05-01 00:00:00
-| 2017-05-01 00:00:00 
+| 2017-05-01 00:00:00
 | defined
 | 2016-03-01 00:00:00
 | 2017-05-01 00:00:00
-| active since #770111
+| active since #770112
```

Comment:
it's #770111 that changes the state to "active", remember that it checks the state for *previous* block to see what rules to enforce. #770112 is the second block that has "active" state, but the first for which the rules are enforced.

I tend to agree that mentining #770112 seems more intuitive, but this is a more subtle distinction than most people take into account :) Very easy to introduce off-by-one errors here.